Sources/Apodini/Response/ApodiniError.swift

```diff
@@ -31,20 +31,29 @@ public struct ApodiniError: Error {
     public typealias Option = AnyPropertyOption<ErrorOptionNameSpace>
     
     private let `type`: ErrorType
-    
+
     private let reason: String?
-    
+
     private let description: String?
+
+    public let information: InformationSet
```

Comment:
This is the same as for `Throws`. I would suggest to make the `InformationSet` part of the `options`.